blog/content/post/2024/dbatools-repl-setup/index.md

```diff
@@ -47,10 +83,22 @@ DistributionServer   : SQL1
 DistributionDatabase : distribution
 ```
 
-(now IsPublisher is true also)
+At this point we have the environment pieces in place in order to create publications, articles and subscriptions.
+
+> Throughout this series of blog posts I'm demonstrating the simplest set of parameters for the commands, but there is more available. As you work through setting up replication in your environment, I'd recommend reading the dbatools docs or running `Get-Help <<command>> -Full` in your PowerShell session to see all the available parameters and examples on how to use them.
+
+For the two commands we've just used you can see the dbatools help documentation here:
+
+- [Enable-DbaReplDistributor](https://dbatools.io/Enable-DbaReplDistributor)
+- [Enable-DbaReplPublishing](https://dbatools.io/Enable-DbaReplPublishing)
+
+## Create Publications
 
+As I mentioned the steps are very similar for whatever flavour of replication you want to create, in this post I'll create a transactional replication publication - this is (in my opinion) the most common type that is used in the real world. Transactional replication allows us to replicate changes from one database to another for certain objects in near real time.
 
-## Add Publication
+The really cool thing here is that this is object level, unlike other technologies like Availability Groups, we can replicate just one table, or even just certain rows from one table based on a filter.
```

Comment:
I suggest adding `columns` too. Like: `we can replicate just one table, some columns, or even just certain rows based on a filter`